### ISSUE TITLE
'ascii' codec can't decode byte 0xe2 in position 2132: ordinal not in range(128)

### ISSUE THREAD

**Issue by w4rn** [Flexget/Flexget#801]
Hello,

i keep having this message and i cant find a solution.
Recently i updated my showrrs link (new website, new link) and from that i have this issue.
```
tasks:
  series:
    rss: http://showrss.karmorra.info/rss.php?user_id=USERID&hd=2&proper=0
    all_series: yes
    pushbullet:
      apikey: *****************
      title: "[F] {{series_name}} {{series_id}}"
      body: "{{title}}\n\nSize: {{content_size}}MB"
    transmission:
      host: localhost
      port: 9091
      username: *********
      password: ********
      path: /media/Delta/Utorrent/Incoming
      addpaused: no
  # sorting task
  sort-files:
    filesystem:
     filesystem:
      # directory with the files to be sorted
      path: /media/Delta/Utorrent/Incoming/
      # fetch all avi, mkv and mp4 files, skips the .part files (unfinished tor$
      regexp: '.*\.(avi|mkv|mp4)$'
    accept_all: yes
    seen: local
    # this is needed for the episode names
    thetvdb_lookup: yes
    all_series:
      # for some reason all_series rejects all episodes here, even with seen: l$
      parse_only: yes
    # TVDB doesn't recognise "Adventure Time with Finn and Jake" so you must ad$
    series:
      - Adventure Time
    move:
      # this is where the series will be put
      to: /media/Delta/{{ tvdb_series_name }}
      # save the file as "Series Name - SxxEyy - Episode Name.ext"
     to: /media/Delta/{{ tvdb_series_name }}
      # save the file as "Series Name - SxxEyy - Episode Name.ext"
      filename: '{{ tvdb_series_name }}.{{ series_id }}.{{ tvdb_ep_name }}{{ lo$
  subtitles:
    filesystem:
      path: /media/Delta/Utorrent/Incoming
      regexp: '.*\.(avi|mkv|mp4)$'
      recursive: yes
    accept_all: yes
    subliminal:
      languages:
        - ita
      alternatives:
        - eng
      exact_match: yes
      providers: [addic7ed, opensubtitles]
      single: no
web_server:
  bind: 0.0.0.0
  port: 5050
api: yes
webui: yes

schedules:
 - tasks: series
   schedule:
    hour: 06-23
    minute: "*/15"
 - tasks: sort-files
   schedule:
    hour: 1-5/2
 - tasks: subtitles
   schedule:
    hour: 06-23
    minute: "*/45"
```

Any idea?

W4rn

**Comment by lepinkainen** [comment]
The actual error message would be useful, with traceback and all.

**Comment by w4rn** [comment]
CRITICAL
plugin
series
'ascii' codec can't decode byte 0xe2 in position 2132: ordinal not in range(128)

Sorry i really forgot :(

**Comment by lepinkainen** [comment]
So there is nothing else than that one line in the log file?

**Comment by w4rn** [comment]
Nope, beside that no other critical error.

**Comment by crawln45** [comment]
That is ambiguous at best other than it's a codec problem. If there is a crash log might want to find it. Otherwise run flexget in debug mode to see if more apparent problem is shown.

**Comment by w4rn** [comment]
Hello,

sorry how can i get a crash log?

**Comment by crawln45** [comment]
If there is one it will be in ~/.flexget

**Comment by w4rn** [comment]
Hello,

here it is:

2016-01-19 17:07 INFO     manager                       Daemonize complete. New PID: 887
2016-01-19 17:07 INFO     scheduler                     Starting scheduler
2016-01-19 17:07 INFO     web_server                    Web interface available at http://127.0.1.1:5050
2016-01-19 17:13 CRITICAL plugin        series          'ascii' codec can't decode byte 0xe2 in position 2132: ordinal not in range(128)
2016-01-19 17:13 WARNING  task          series          Aborting task (plugin: rss)
2016-01-19 17:13 ERROR    entry         sort-files      Failed the.big.bang.theory.908.hdtv-lol (Path value replacement `/media/Delta/{{ tvdb_series_name }}` failed: (UndefinedError) 'tvdb_series_name' is undefined)
2016-01-19 17:13 INFO     task          sort-files      Plugin move has requested task to be ran again after execution has completed.
2016-01-19 17:13 ERROR    entry         sort-files      Failed the.originals.306.hdtv-lol (Path value replacement `/media/Delta/{{ tvdb_series_name }}` failed: (UndefinedError) 'tvdb_series_name' is undefined)
2016-01-19 17:13 ERROR    entry         sort-files      Failed the.vampire.diaries.706.hdtv-lol (Path value replacement `/media/Delta/{{ tvdb_series_name }}` failed: (UndefinedError) 'tvdb_series_name' is undefined)
2016-01-19 17:13 ERROR    entry         sort-files      Failed Franny.2015.iTALiAN.MD.WEBDL.XviD-M76 (location `/media/Delta/Utorrent/Incoming/Franny.2015.iTALiAN.MD.WEBDL.XviD-M76.avi` does not exists (anymore).)
2016-01-19 17:13 ERROR    entry         sort-files      Failed Revenant.Redivivo.2015.iTALiAN.MD.720p.HDSCR.x264-LL1 (location `/media/Delta/Utorrent/Incoming/Revenant.Redivivo.2015.iTALiAN.MD.720p.HDSCR.x264-LL1.mkv` does not exists (anymore).)
2016-01-19 17:13 INFO     task          sort-files      Rerunning the task in case better resolution can be achieved.
2016-01-19 17:15 CRITICAL plugin        series          'ascii' codec can't decode byte 0xe2 in position 2132: ordinal not in range(128)
2016-01-19 17:15 WARNING  task          series          Aborting task (plugin: rss)
2016-01-19 17:30 CRITICAL plugin        series          'ascii' codec can't decode byte 0xe2 in position 2132: ordinal not in range(128)
2016-01-19 17:30 WARNING  task          series          Aborting task (plugin: rss)
2016-01-19 17:45 CRITICAL plugin        series          'ascii' codec can't decode byte 0xe2 in position 2132: ordinal not in range(128)
2016-01-19 17:45 WARNING  task          series          Aborting task (plugin: rss)
2016-01-19 17:45 INFO     subtitles     subtitles       Name computed for /media/Delta/Utorrent/Incoming/shameless.us.602.hdtv-lol.mp4 was shameless (US)
2016-01-19 17:47 ERROR    entry         subtitles       Failed shameless.us.602.hdtv-lol (cannot find any subtitles for now.)
2016-01-19 17:47 INFO     task          subtitles       Plugin subliminal has requested task to be ran again after execution has completed.
2016-01-19 17:47 INFO     subtitles     subtitles       Name computed for /media/Delta/Utorrent/Incoming/supergirl.110.hdtv-lol.mp4 was supergirl
2016-01-19 17:47 ERROR    entry         subtitles       Failed supergirl.110.hdtv-lol (cannot find any subtitles for now.)
2016-01-19 17:47 INFO     task          subtitles       Rerunning the task in case better resolution can be achieved.
2016-01-19 18:00 CRITICAL plugin        series          'ascii' codec can't decode byte 0xe2 in position 2132: ordinal not in range(128)
2016-01-19 18:00 WARNING  task          series          Aborting task (plugin: rss)
2016-01-19 18:15 CRITICAL plugin        series          'ascii' codec can't decode byte 0xe2 in position 2132: ordinal not in range(128)
2016-01-19 18:15 WARNING  task          series          Aborting task (plugin: rss)
2016-01-19 18:30 CRITICAL plugin        series          'ascii' codec can't decode byte 0xe2 in position 2132: ordinal not in range(128)
2016-01-19 18:30 WARNING  task          series          Aborting task (plugin: rss)

**Comment by liiight** [comment]
This is the regular log. There should also be crash log report.
